app/pages/admin/organization-status.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+import apiClient from 'panoptes-client/lib/api-client';
+import moment from 'moment';
+
+import ProjectIcon from '../../components/project-icon';
+import LoadingIndicator from '../../components/loading-indicator';
+
+import Toggle from './project-status/toggle';
+
+class OrganizationStatus extends Component {
+  constructor(props) {
+    super(props);
+    this.forceUpdate = this.forceUpdate.bind(this);
```

Comment:
Oh, thought that's a no go for new code. Is that necessary as a`ChangeListener` replacement?